app/grandchallenge/algorithms/views.py

```diff
@@ -478,15 +479,11 @@ def get_success_url(self):
         return self.algorithm.get_absolute_url()
 
 
-class JobCreate(
+class JobCreatePermissionMixin(
```

Comment:
Unsure about the permissions implementation here. I used what we currently have on the `JobCreateView` and put it into a mixin for re-use across both views. But it uses `get_permission_object`, so we probably want to change the way we check permissions. We could use the `AccessMixin`, but then we have the issue that the `VerificationRequiredMixin` already subclasses that, which might be problematic. Is the best way forward then to drop the verification mixin and include the verification check in the dispatch method of this new mixin here?

Reply:
`VerificationRequiredMixin` uses `UserPassesTestMixin` which cannot be mixed with other `UserPassesTestMixin`. That is due to `test_func` only being able to be defined once as the base raises `NotImplementedError` so `super()` cannot be reliably used, meaning multiple `test_func`s would be skipped.

If `VerificationRequiredMixin` used `AccessMixin` it can be mixed with other `AccessMixin`s. That is because the `dispatch` method that it overrides calls `super()`.

Reply:
Alright, then I will change the `VerificationRequiredMixin`. I will do that in a separate PR to main.